### ISSUE TITLE
Ability to Resize Window

### ISSUE THREAD

**Issue by theundeadlich** [flipperdevices/qFlipper#191]
Could we have the ability to resize the qFlipper window for those of us on higher or different resolution screens so that we can see it better.

**Comment by zhovner** [comment]
Can you please show how it looks on your screen?

**Comment by theundeadlich** [comment]
Sure thing!

Here it is on 3840x2160

<img width="1919" alt="image" src="https://github.com/flipperdevices/qFlipper/assets/55335801/f8a70b29-4624-46b3-90f3-f9762c4a6124">

**Comment by zhovner** [comment]
Hm, it should scale the window automatically, isnt? @gsurkov

**Comment by bwachter** [comment]
Screenshot from the other side (6" 1280x720 display):

![qFlipper_small_screen](https://github.com/flipperdevices/qFlipper/assets/767461/ae9b5e10-c5ee-44be-b182-60e60d345b58)

**Comment by vtrenton** [comment]
+1 on this - when using tiling windowing managers or generally just snapping windows to maximize screen real estate value the qflipper app refuses to conform breaking most tiling flows :(

**Comment by theundeadlich** [comment]
I am able to make the application slightly bigger by using High DPI Scaling Override and setting it to system, but this still doesn't allow us to rescale the window.

<img width="1920" alt="image" src="https://github.com/flipperdevices/qFlipper/assets/55335801/3f2a80b1-e843-42d6-a607-8f27bbce81c9">

**Comment by gsurkov** [comment]
TL;DR:
qFlipper does its best to guess what the window size should be according to the system settings. In case it fails to do so, it's possible to instruct it by setting the environment variable `QT_SCALE_FACTOR` to what feels best. Make sure not to set it globally as it will affect other Qt-based applications, too.

A bit more trivia:
1. qFlipper uses a fixed-size window by design. It was a deliberate decision to make it look like the janky early 2000s programs with custom windows. We might rethink this decision for the version 2.0 as it causes more problems than anticipated (people have stopped designing apps this way for a reason).
2. qFlipper reads HiDPI settings from the underlying OS and adjusts automatically thanks to the Qt library. It doesn't look at the resolution or physical screen size, but at the scaling ratio set in the system. Might add an explicit setting if it doesn't work well automatically.
